recipe/meta.yaml

```
@@ -36,11 +40,15 @@ test:
 
 about:
   home: https://pagure.io/python-daemon/
+  description: |
+    This library implements the well-behaved daemon specification of
+    :pep:`3143`, “Standard daemon process library”.
```

```suggestion
    PEP 3143, “Standard daemon process library”.
```
to avoid any unexpected things on anaconda.org UI